install.html

```diff
@@ -7,9 +7,85 @@
 <script src=docs.js></script>
 
 <h1>Selenium Installation</h1>
+Selenium set up is quite different than other commercial tools. You don't install a tool using
+selenium binaries but set up your test automation project to use selenium binary in the langauge
+of your choice. The selenium binary you would need depends on browsers you want to run
+your tests upon.
 
 
+<h2>Using browser specific jar</h2>
+If you want to run tests in only a spcific brower then you can add browser specific dependency in
+your pom.xml file. You should add following dependency in your pom.xml file to run
+tests only on Firefox browser -
+
+
+<textarea rows="6" cols="100" style="border:none; margin-top:10px; margin-bottom:10px;">
```

Comment:
Because it is the appropriate tag to use. If you are missing the tags, you will have to escape them with the `&lt;` and `&gt;` entities.